src/encoded/docs/schema-changes.md

```diff
@@ -154,69 +154,141 @@ This new object is an array of example objects that can successfully POST agains
+
+* Examples include:
+    1) Changing the name of a property in the existing schema.
+    2) Removing a property from the existing schema.
+    3) A property that previously allowed free text is now changed to a possible list of allowed enums.
+    4) If an existing enum is removed from a property.
+
+
+* Most of the cases described above are examples where there will be a potential conflict for all the existing objects to be validated under the new schema. Hence, an additional step of adding an upgrader script will be required. This will ensure that all the existing objects will be changed such that they can fit into the new schema that is currently being implemented.
+
+* One more case needs a mention here: When adding multiple new schema properties that would lead to substantial changes within an existing schema, one must update the schema version. Even though, the addition of multiple new schema properties is not going to invalidate any existing objects, it would be useful to do so. Particularly, this will be helpful to all the submitters and users who are trying to use these properties either to submit their data or while querying the database using scripts.
+
+**Follow the steps as outlined below**
+
+1. In the **schemas** directory, edit the existing properties in the corresponding JSON file named after the object and increment the schema version. 
+
+**Specific example from the genetic modifications object schema change:**
+
+Up until schema version 6 for the genetic modifications object, one of the possible enums for "purpose" property was "validation". As a part of schema version upate to 7, "validation" was removed and instead the term "characterization" was added.
+
 
-1. In the **schemas** directory, increment the schema version (1->2), if it meets the above rule: 
+        "purpose":{
+            "title": "Purpose",
+            "description": "The purpose of the genetic modification.",
+            "type": "string",
+            "enum": [
+                "activation",
+                "analysis",
+                "overexpression",
+                "repression",
+                "tagging",
+                "validation",
+                "screening",
+                "expression"
+            ]
+        },
+        # Replacing the enum "validation" by the enum "characterization" in the list of enums of the "purpose" property:
+        "purpose":{
+            "title": "Purpose",
+            "description": "The purpose of the genetic modification.",
+            "type": "string",
+            "enum": [
+                "activation",
+                "analysis",
+                "overexpression",
+                "repression",
+                "tagging",
+                "characterization",
+                "screening",
+                "expression"
+            ]
+        },
+
```

Comment:
"schema_version": {
            "default": "6"
        },
 "purpose":{
            "title": "Purpose",
            "description": "The purpose of the genetic modification.",
            "type": "string",
            "enum": [
                "activation",
                "validation",
                "screening"                
            ]
        },
        # Replacing the enum "validation" by the enum "characterization" in the list of enums of the "purpose" property:
"schema_version": {
            "default": "7"
        },        
"purpose":{
            "title": "Purpose",
            "description": "The purpose of the genetic modification.",
            "type": "string",
            "enum": [
                "activation",
                "characterization",
                "screening"
            ]
        },